repository/src/test/resources/alfresco/local-transform-service-config-pipeline-test.json

```diff
@@ -34,9 +34,10 @@
     {
```

Comment:
Modifications to get tests to pass, now that we throw out pipelines with invalid steps.
Had to add in the missing steps, so the transformers count goes up bu two.